package/npm/package.json

```diff
@@ -1,14 +1,12 @@
 {
   "name": "elm-format",
-  "version": "0.8.5",
+  "version": "0.8.6-0",
```

Comment:
We learned in https://github.com/mpizenberg/elm-test-rs/pull/121#issuecomment-1374454965 that it usually takes a few tries publishing everything correctly the first time, so I went ahead and added `-0` to everything. Once it’s working, we can release the top package as just `0.8.6`. Read about versioning of the sub packages here: https://github.com/elm/compiler/pull/2287/files#diff-cd34b117c7124e742fd6eb9ffd8f6f9f5c125402a66e9e62c906785b940f7681R68-R85

Reply:
Ah, okay, so if I'm understanding correctly:

- binary packages will always be name=`@avh4/elm-format` version=`$OS-$ARCH-$MAJ.$MIN.$PAT-$BINVER`
- for pre-release versions, the base package will be name=`elm-format` version=`$MAJ.$MIN.$PAT-$PREVER`
- once finalized, the base package will be name=`elm-format` version=`$MAJ.$MIN.$PAT`

Reply:
I would put it like this:

- The base package follows regular npm semver
- The binary packages don’t matter at all how you version, except for your own sanity
- The binary packages have the name `@avh4/elm-format-$OS-$ARCH`